app/views/employees/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Add new employee</h1>
+
+<%= form_for @employee do%>
+  <div>
+        <label for="name">Name:</label>
+        <input placeholder="Exam. Luis Perez" type="text" name="name" value="<%= @employee.name %>"/>
+    </div>
+    <div>
+        <label for="email">Email:</label>
+        <input type="email" placeholder="example@example.com" name="email" value ="<%= @employee.email%>"/>
+    </div>
+    <div>
+        <label for="position">Position:</label>
+        <input type="text" placeholder="example@example.com" name="position" value="<%= @employee.position %>"/>
+    </div>
+
+    <div>
+        <label for="private_number">Private Number:</label>
+        <input type="password" placeholder="Exam: 123456" name="private_number" value ="<%= @employee.private_number%>"/>
+    </div>
+    <button >Edit Employee</button>
+    
```

Comment:
Please fix the indentation of this file at 2 spaces

Reply:
Done